packages/browser-wallet-api-helpers/package.json

```
@@ -0,0 +1,35 @@
+{
+    "name": "@concordium/browser-wallet-api-helpers",
+    "version": "0.1.0",
+    "license": "Apache-2.0",
+    "packageManager": "yarn@3.2.0",
+    "main": "lib/index.js",
```

```suggestion
    "main": "lib/index.js",
    "browser": "lib/concordiumHelpers.min.js"
```
At least is what I do in the web-sdk :shrimp: